BadCodeV3.php

```
@@ -0,0 +1,67 @@
+        if ($userId == 'VIP') {
+            $total = $total * 0.8; // 20%折扣
+        }
+        
+        // 硬编码的配置数据
+        $shippingCost = 15.99;
+        $taxRate = 0.085;
+        
+        $finalTotal = $total + $shippingCost + ($total * $taxRate);
+        
+        // SQL注入风险
+        $sql = "INSERT INTO orders (user_id, total) VALUES (" . $userId . ", " . $finalTotal . ")";
```

PHPMD (2.15.0)

Avoid unused local variables such as '$sql'. (Unused Code Rules)

(UnusedLocalVariable)